packages/agoric-cli/package.json

```diff
@@ -39,12 +39,12 @@
     "@agoric/swingset-vat": "^0.30.2",
     "@agoric/zoe": "^0.25.3",
     "@confio/relayer": "^0.7.0",
-    "@cosmjs/crypto": "0.29.0",
-    "@cosmjs/encoding": "0.29.0",
-    "@cosmjs/math": "0.29.0",
-    "@cosmjs/proto-signing": "0.29.0",
-    "@cosmjs/stargate": "0.29.0",
-    "@cosmjs/tendermint-rpc": "0.29.0",
+    "@cosmjs/crypto": "^0.29.0",
```

Comment:
+1 for `^` but please just go ahead and bump the version numbers so it's more obvious what version is being imported